clients/drcachesim/drpt2trace/pt2ir.cpp

```diff
@@ -243,7 +247,6 @@ pt2ir_t::convert(OUT instrlist_t **ilist)
             if (status == -pte_eos)
                 break;
             dx_decoding_error(status, "sync error", insn.ip);
-            instrlist_clear_and_destroy(GLOBAL_DCONTEXT, *ilist);
```

Comment:
The destructor of a pass-by-reference parameter is *not* called when the function exits, so this cannot be removed.  Ditto below.

Reply:
Done.